### PR TITLE
Implement community board (categories and org lists) [6/7]

### PR DESCRIPTION
**Pull request by isaachan100**
## Overview

Implement community board categories and org lists

## Changes Made

- updated organization entity with category field
- added query in OrganizationResolver: getOrganizationsByCategory(categorySlug: String!): [organization]! 
- updated flyer models to allow each flyer to be associated with a list of organizations and a list of organization slugs
- updated FlyerRepo and OrganizationRepo to account for changes in flyer model
- updated unit jest testing to reflect changes

## Test Coverage

- tested getOrganizationByCategory query locally
- jest testing runs to completion

## Next Steps (delete if not applicable)

- finish onboarding organizations and then we're done!!! :)

### CODE REVIEW

File: src/resolvers/OrganizationResolver.ts [hunk 1]
```diff
@@ -45,17 +53,17 @@ class OrganizationResolver {
   }
 
   @FieldResolver((_returns) => Number, {
-    description: "The total shoutouts of an <Organization's> <Flyers>",
+    description: 'The total number of <Flyers> from an <Organization>',
```
Comment (katesliang) [inline src/resolvers/OrganizationResolver.ts:56]: nit: keep description consistent as other ones (e.g. "Returns the total # ...")

File: src/resolvers/OrganizationResolver.ts [hunk 2]
```diff
@@ -45,17 +53,17 @@ class OrganizationResolver {
...
   })
-  async shoutouts(@Root() organization: Organization): Promise<number> {
-    return OrganizationRepo.getShoutouts(organization);
+  async numFlyers(@Root() organization: Organization): Promise<number> {
+    return OrganizationRepo.getNumFlyers(organization);
   }
 
   @FieldResolver((_returns) => Number, {
-    description: 'The total number of <Flyers> from an <Organization>',
+    description: "The total shoutouts of an <Organization's> <Flyers>",
```
Comment (katesliang) [inline src/resolvers/OrganizationResolver.ts:63]: same as above

**Review by katesliang — APPROVED** [review]
insanely prod

**Review by snajima — CHANGES_REQUESTED** [review]

File: src/resolvers/OrganizationResolver.ts (hunk 1 above)
Comment (isaachan100) [reply src/resolvers/OrganizationResolver.ts]: prod sg

File: src/app.ts [hunk 3]
```diff
@@ -20,6 +20,14 @@ import WeeklyDebriefRepo from './repos/WeeklyDebriefRepo';
...
 const main = async () => {
   const schema = await buildSchema({
+    resolvers: [
+      ArticleResolver,
+      FlyerResolver,
+      MagazineResolver,
+      OrganizationResolver,
+      PublicationResolver,
+      UserResolver,
+    ],
```
[the remark this answers is not in the document]
Comment (isaachan100) [reply src/app.ts]: yeah ur right i deleted them

**Review by snajima — APPROVED** [review]
super based

**Review by katesliang — APPROVED** [review]

**Pull request merged**